articles/synapse-analytics/spark/synapse-file-mount-api.md

```diff
@@ -80,8 +80,22 @@ mssparkutils.fs.mount(
 > [!NOTE]   
 > You might need to import `mssparkutils` if it's not available: 
 > ```python
-> From notebookutils import mssparkutils 
-> ``` 
+> from notebookutils import mssparkutils 
+> ```
+> Mount parameters:
+> 1. fileCacheTimeout: Blobs will be cached in the local temp folder for 120 seconds by default. During this time, blobfuse will not check whether the file is up to date or not. The parameter could be set to change the default timeout time. When multiple clients modify files at the same time, in order to avoid inconsistencies between local and remote files, It's recommended to shorten the cache time, or even change it to 0, and always get the latest files from the server.
```

Comment:
```suggestion
> - fileCacheTimeout: Blobs will be cached in the local temp folder for 120 seconds by default. During this time, blobfuse won't check whether the file is up to date or not. The parameter could be set to change the default timeout time. When multiple clients modify files at the same time, in order to avoid inconsistencies between local and remote files, we recommend shortening the cache time, or even changing it to 0, and always getting the latest files from the server.
```